modules/browser/src/main/resources/org/gjt/jclasslib/browser/messages/Browser_pl.properties

```diff
@@ -225,86 +225,88 @@ message.invalid.constant.pool.reference=niepoprawna referencja do puli stałych
-action.attach.to.jvm.description=Attach to a running JVM
-action.detach.from.jvm=Detach From JVM
-action.detach.from.jvm.description=Detach from the currently attached JVM
-window.select.running.jvm=Select Running JVM
-message.attach.failed.0=Attach failed: {0}
-message.management.agent.error.0=Could not load management agent: {0}
-message.connection.failed.0=Connection failed: {0}
-message.could.not.redefine.class.file=Could not redefine class file
-error.message.was.0=The error message was: {0}
-classes.attach.info=Note: Classes that are loaded in attach mode are reconstituted by the JVM and and only contain information that is relevant at runtime
+menu.class=Klasa
+access.flag.since.0=dostępna od {0}
+action.show.jvm.spec=Otwórz specyfikację JVM
+action.replace.opcode=Zamień opcode
+choose.compatible.opcode=Wybierz kompatybilny opcode, który ma taki sam rozmiar i modyfikuje stos w ten sam sposób:
+replace.opcode.title=Zamień opcode
+key.byte.value=Wartość całkowitoliczbowa (byte):
+input.not.valid.byte=Wprowadzona wartość nie jest poprawną liczbą typu byte
+key.short.value=Wartość całkowitoliczbowa (short):
+input.not.valid.short=Wprowadzona wartość nie jest poprawną liczbą typu short
+key.integer.value=Wartość całkowitoliczbowa (integer):
+input.not.valid.integer=Wprowadzona wartość nie jest poprawną liczbą typu integer
+action.edit.immediate.byte=Edytuj stałą bezpośrednią (byte)
+action.edit.index=Edytuj indeks
+action.edit.immediate.short=Edytuj stałą bezpośrednią (short)
+action.edit.constant=Edytuj stałą
+action.edit.branch.offset=Edytuj offset gałęzi
+action.edit.dimensions=Edytuj liczbę wymiarów
+action.edit.count=Edytuj count
+action.jump.offset.0=Offset skoku {0}
+action.edit.jump.offsets=Edytuj offsety skoków
+action.jump.offset.for.match.0=Offset skoku dla dopasowania {0}
+action.match.0=Dopasowanie {0}
+action.edit.matches=Edytuj dopasowania
+action.edit.default.jump.offset=Edytuj domyślny offset skoku
+action.add.export=Dodaj exports
+action.add.opens=Dodaj opens
+enter.package.name=Nazwa paczki:
+message.class.not.loaded=Klasa {0} nie została załadowana.
+action.attach.to.jvm=Podłącz się do działającej JVM
+action.attach.to.jvm.description=Podłącz się do działającej JVM
+action.detach.from.jvm=Odłącz się od JVM
+action.detach.from.jvm.description=Odłącz się od JVM, do której aktualnie jesteś podłączony
+window.select.running.jvm=Wybierz działającą JVM
+message.attach.failed.0=Próba podłączenia zakończona niepowodzeniem: {0}
+message.management.agent.error.0=Nie można załadować management agenta: {0}
+message.connection.failed.0=Połączenie nieudane: {0}
+message.could.not.redefine.class.file=Nie udało się zredefiniować pliku klasy
+error.message.was.0=Treść błędu: {0}
+classes.attach.info=Uwaga: Klasy załadowane po podłączeniu się do JVM są odtworzone przez JVM i zawierają tylko informacje istotne w czasie wykonywania programu
+no.compatible.opcode=Brak kompatybilnych opcode\'ów dla wybranej instrukcji
```

Comment:
I'm not sure how to type an apostrophe, both `'` and `\'` work, but GitHub highlights them both wrong

Reply:
Apostrophes do not need be quoted in this case, I would not worry about the GitHub highlighting